src/main/java/org/terasology/structureTemplates/util/transform/HorizontalBlockRegionRotation.java

```diff
@@ -79,7 +79,7 @@ public Block transformBlock(Block block) {
             return sideDefinedBlockFamily.getBlockForSide(transformSide(block.getDirection()));
         } else if (blockFamily instanceof AttachedToSurfaceFamily) {
             // TODO add some proper method to block famility to not have to do this hack
-            return blockFamily.getBlockForPlacement(null, null, null, transformSide(block.getDirection()), null);
```

Comment:
this line should be:
``` return blockFamily.getBlockForPlacement(null,transformSide(block.getDirection()), null); ```

look at the original code and compare it to the new block family.
https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/world/block/family/HorizontalBlockFamily.java#L63
https://github.com/MovingBlocks/Terasology/blob/newBlockFamilies/engine/src/main/java/org/terasology/world/block/family/HorizontalFamily.java#L90